modules/rpc/yarpc_test.go

```diff
@@ -36,30 +35,36 @@ func TestRegisterDispatcher_OK(t *testing.T) {
 	RegisterDispatcher(defaultYARPCDispatcher)
 }
 
+func TestRegisterStarter_OK(t *testing.T) {
+	t.Parallel()
+	RegisterStarter(defaultYARPCStarter)
```

Comment:
it is actually effective once you'll start checking for races.
I am very sad we don't use t.Parallel everywhere.